runner.yml.dist

```diff
@@ -0,0 +1,43 @@
+drupal:
+  root: "build"
+  base_url: "http://web:8080/build"
+  site:
+    name: "OpenEuropa"
+    profile: "minimal"
+  database:
+    host: "mysql"
+    port: "3306"
+    name: "oe_oembed"
+    user: "root"
+    password: ""
+  post_install:
+    # Prepare the instance.
+    - "./vendor/bin/drush en toolbar -y"
```

Comment:
Should we use the `admin_toolbar` instead? I can update the `drupal-module-template` if so.

Reply:
No. That is in EWCMS.